pages/profile-setup/components/ProfileSetupForm/ProfileSetupForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { useRouter } from "next/router";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { AxiosError } from "axios";
+import classNames from "classnames/bind";
+import { useForm } from "react-hook-form";
+
+import { postProfileSetup } from "@/apis/apis";
+import Input from "@/components/Input";
+import { PAGE_PATH } from "@/constants/pagePath";
+
+import FormSchema from "./FormSchema";
+import styles from "./ProfileSetupForm.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ProfileSetupFormProps {
+  accountId: number;
+  email: string;
+}
+
+export interface FieldValuesType {
+  nickname: string;
+  phoneNumber: string | null;
+}
+
+export default function ProfileSetupForm({ accountId, email }: ProfileSetupFormProps) {
+  const router = useRouter();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FieldValuesType>({
+    mode: "all",
+    defaultValues: { nickname: "", phoneNumber: null },
+    resolver: zodResolver(FormSchema),
+  });
+
+  const onSubmit = async (data: Omit<FieldValuesType, "accountId">) => {
+    const formData = { ...data, accountId: Number(accountId) };
+
+    try {
+      await postProfileSetup(formData);
+      router.push(PAGE_PATH.MAIN);
```

Comment:
```suggestion
      router.push(PAGE_PATH.DASHBOARD);
```
p3: 추가 정보까지 입력하면 대시보드 페이지로 가야할 것 같아요!

Reply:
헉 그러네용...! 수정하겠습니다!!!